querybook/server/app/auth/__init__.py

```diff
@@ -1,14 +1,32 @@
+import time
+
 from flask import request
 from flask_login import logout_user, current_user
 
 from const.path import BUILD_PATH
 from const.datasources import DS_PATH
 from env import QuerybookSettings
 from lib.utils.import_helper import import_module_with_default
+from lib.stats_logger import stats_logger, ACTIVE_USER_COUNTER
 
 
 auth = None
 login_config = None
+active_users = {}
```

Comment:
this wouldn't work if you have multiple servers, unless you plan to store this stat in redis